Assets/tiny_rogue/Scripts/Systems/GameViewSystem.cs

```diff
@@ -37,6 +37,24 @@ static float GetAlphaForStaticTile(Tile tile)
             return color;
         }
         
+        static Color GetColorForDoor(Tile tile)
```

Comment:
Not quite sure I understand why this needs to be used intead of `GetAlphaForStaticTile`, ie. why do doors specifically want to be grey RGB not alpha?

Reply:
They look transparent as they're on top of a tile

Reply:
![image](https://user-images.githubusercontent.com/9428660/60286453-dcd4bf00-990f-11e9-9009-ae83d86076b7.png)
Example

Reply:
Ahh, touche